source/Calamari/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -246,13 +247,13 @@ Log-VersionTable
 # -----------------------------------------------------------------
 # Variables
 # -----------------------------------------------------------------
-
+{{BeforeVariablesDebugLocation}}
 {{VariableDeclarations}}
```

Comment:
Was debating with myself whether it was overkill having these...
But... The ability for us to debug our stuff is super helpful too. Maybe we just dont call these options out in the documentation.